koku/api/settings/utils.py

```diff
@@ -2,10 +2,18 @@
 # Copyright 2021 Red Hat Inc.
 # SPDX-License-Identifier: Apache-2.0
 #
+import typing as t
+
+import django.core.exceptions
+import django_filters
+import rest_framework.exceptions
```

Comment:
Why are we not importing what we need and use? We almost always import specifics, not entire modules, such as:
```
from rest_framework.exceptions import ValidationError
from django_filters.filters import MultipleChoiceFilter
from django.core.exceptions import ValidationError as DjangoValidationError
```